app/models/menu.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Menu < ApplicationRecord
+  belongs_to :user
+  has_one :review
+  belongs_to :dish
+
+  enum meal_type: { lunch: 0, snack: 1 }
```

Comment:
why are we storing meal_type in both Menu and MealAttendance?

Reply:
It is because we need to manage which meal a menu belongs to and a user giving which meal's attendance.